src/core/initializeComponentsFactory.js

```diff
@@ -44,7 +42,10 @@ export default (componentCreators, logger, getNamespacedStorage) => config => {
     componentRegistry.register(namespace, component);
   });
 
-  // TODO: Output the finalized config schema (if debug is turned on)
+  // Output the finalized configuration
+  if (config.debug) {
+    logger.info(JSON.stringify(config, null, 2));
```

Comment:
We should probably add a label on this message.

Reply:
True, I added this to present the finalized configuration. The same could be done with the schema but I want to know if this would be the correct way to present (just when debug is enabled) or if there should be a specific command to do so.

Reply:
You don't need to wrap `logger.info` in the `if` statement. The logger internally logs or doesn't log based on whether debugging is enabled.